tests/dummy/app/router.js

```diff
@@ -8,21 +8,27 @@ const Router = AddonDocsRouter.extend({
 
 Router.map(function() {
   docsRoute(this, function() {
+    this.route('installation');
+    this.route('quickstart');
+    this.route('deploying');
+
+    this.route('dependencies');
+    this.route('isolation');
```

Comment:
I think core concepts continue better than isolation.

Reply:
Core concepts is it's own section, this is just a file.